portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,14 +5,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
-/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+/**
+ * Handles requests sent to the /hello URL. Try running a server and navigating
+ * to /hello!
+ */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
-}
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        ArrayList<String> arr = new ArrayList<String>(); // Creates an arrayList
+        // Add Elements to the list
+        arr.add("Well, you have found my flabbergast button, and guess what? You pressed it. -Schmidt, New girl");
+        arr.add("I Was Never One To Hold A Grudge, Jeffrey. My Father Held Grudges. Ill Always Hate Him For That. -Pierce, Community");
+        arr.add("My first girlfriend turned into the moon. That's rough, buddy. -Sokka & Zuko, Avatar the Last Airbender");
+        arr.add("Its gonna be legen, wait for it and I hope you are not lactose-intolerant because the second half of that word is dary -Barney, How I met your Mother");
+        arr.add("Stuff can be two things. -Detective Jake Peralta, Brooklyn Nine-Nine");
+        // convert to json string
+        String json = convertToJsonUsingGson(arr);
+
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
+    }
+
+    private String convertToJsonUsingGson(ArrayList<String> arr) {
+        Gson gson = new Gson();
+        String json = gson.toJson(arr);
+        return json;
+    }
```

Comment:
optional 🤷 : no need here to capture the response in a variable if you immediately return it.

Exs:

`return gson.toJson(data);` or even keep a `Gson` object as a static field on the Servlet like 

```
public class FactServlet extends HttpServlet {
  public static final Gson gson = new Gson();

@Override  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
  //...
  String json = gson.toJson(data);
```
which allows you to get rid of the function altogether.


Side note: Don't let me discourage you from using variables like this. Local variables, especially non-necessary ones, are great for making your code more readable. A toy example:

```
// before 
if (clock.getHours() > 12 && clock.isPm() && clock.getHours() < 5) {
  return true;
}
...
```

```
// after
boolean isAfterNoon = clock.getHours() > 12 && clock.isPm() && clock.getHours() < 5;

if (isAfternoon) {
  return true;
}
```